modules/op-release-notes-1-8.adoc

```diff
@@ -36,22 +285,179 @@ For the *Tekton Operator*, the `operator.tekton.dev/v1alpha1` API version is *no
+// link:https://issues.redhat.com/browse/SRVKP-1539
+// Shivam Mukhade
+
+// .Pipelines
+
+* In {pipelines-title} 1.8, the `PipelineRun.Status.TaskRuns` and `PipelineRun.Status.Runs` fields are deprecated and are planned to be removed in a future release. See link:https://github.com/tektoncd/community/blob/main/teps/0100-embedded-taskruns-and-runs-status-in-pipelineruns.md[TEP-0100: Embedded TaskRuns and Runs Status in PipelineRuns].
+
+* In {pipelines-title} 1.8, the `pipelineRunCancelled` state is deprecated and planned to be removed in a future release. Graceful termination of `PipelineRun` objects is now promoted from an alpha feature to a stable feature. (See link:https://github.com/tektoncd/community/blob/main/teps/0058-graceful-pipeline-run-termination.md[TEP-0058: Graceful Pipeline Run Termination].) As an alternative, you can use the `Cancelled` state, which replaces the `pipelineRunCancelled` state.
++
+You do not need to make changes to your `Pipeline` and `Task` resources. If you have tools that cancel pipeline runs, you must update tools in the next release. This change also affects tools such as the CLI, IDE extensions, and so on, so that they support the new `PipelineRun` statuses.
+
+* In {pipelines-title} 1.8, the `timeout` field in `PipelineRun` has been deprecated. Instead, use the `PipelineRun.Timeouts` field, which is now promoted from an alpha feature to a stable feature.
+// link:https://github.com/tektoncd/pipeline/pull/4813
+
+* In {pipelines-title} 1.8, `init` containers are omitted from the `LimitRange` object's default request calculations.
+// link:https://github.com/tektoncd/pipeline/pull/4769
+
+
 [id="known-issues-1-8_{context}"]
 == Known issues
 
 * The `s2i-nodejs` pipeline cannot use the `nodejs:14-ubi8-minimal` image stream to perform source-to-image (S2I) builds. Using that image stream produces an `error building at STEP "RUN /usr/libexec/s2i/assemble": exit status 127` message.
 +
-Workaround: Use `nodejs:14-ubi8` instead of the `nodejs:14-ubi8-minimal` image stream.
+Workaround: Use `nodejs:14-ubi8` rather than the `nodejs:14-ubi8-minimal` image stream.
 // (link:https://issues.redhat.com/browse/SRVKP-1782[#SRVKP-1782])
 // Writer, also see https://github.com/tektoncd/operator/blob/main/cmd/openshift/operator/kodata/tekton-addon/addons/02-clustertasks/source_local/s2i-nodejs/s2i-nodejs-task.yaml. Overkill: The user can run the `$ oc edit clustertask s2i-nodejs` command and replace `$(params.VERSION)` with `14-ubi8`. Alternatively, you can use a `TaskRun` resource to set the tag of the `nodejs` image stream to `14-ubi8`.
 
-// [id="fixed-issues-1-8_{context}"]
-// == Fixed issues
+// .Operator
+* The `s2i-nodejs` image cannot use the `nodejs:14-ubi8-minimal` image stream to perform S2I builds. Doing so causes the following error message: `error building at STEP "RUN /usr/libexec/s2i/assemble": exit status 127`. There is no workaround for this issue.
```

Comment:
@sm43 This release note and the one on line 314 are duplicates, but differ on whether there is a workaround. Which one should we keep?